models/demos/deepseek_v3/tests/test_mla_1d.py

```diff
@@ -34,98 +32,11 @@ def hf_config_short(hf_config):
 def reference(hf_config_short, reset_seeds):
     """Get the actual DeepSeek MLA model using local implementation."""
 
-    model = DeepseekV3Attention(hf_config_short, layer_idx=0)
+    model = DeepseekV3Attention(hf_config_short, layer_idx=0).eval()
```

Comment:
if you're converting the reference models to eval, please do so in the other tests as well. Let's not leave this to someone who notices this discrepancy later.